roles/matrix-bridge-sms/defaults/main.yml

```diff
@@ -26,7 +26,7 @@ matrix_sms_bridge_systemd_wanted_services_list: []
 
 matrix_sms_bridge_appservice_url: 'http://matrix-sms-bridge:8080'
 matrix_sms_bridge_homeserver_hostname: 'matrix-synapse'
-matrix_sms_bridge_homeserver_port: '8008'
+matrix_sms_bridge_homeserver_port: "{{ matrix_synapse_container_client_api_port }}"
```

Comment:
We should move this to `group_vars/matrix_servers`. You can define `matrix_sms_bridge_homeserver_port` as `0` here and ensure it's set via `roles/matrix-bridge-sms/tasks/validate_config.yml `. Maybe a default value of null (`~` in YAML) makes validating this easier.

Reply:
Ok will replace it with `~`